### PR TITLE
edit of AGMS317

### PR DESCRIPTION
**Pull request by finp**

### CODE REVIEW

File: modules/ROOT/pages/_partials/proc_self-signed-certs.adoc [hunk 1]
```diff
@@ -43,77 +42,91 @@ The resulting file is located in the `/tmp` directory.
...
+. Copy the file to your device using one of the following methods:
 +
-* Email attachment (Recommended) - Simply email the certificate to an address
-accessible from the device and download the attachment.
-* Cloud service - Use a cloud storage service such as Dropbox or Google
-Drive which is accessible from the device, browse to the certificate and
-proceed to download it.
-* USB - Attach the device to the machine via USB and drag the certificate to a devices file system. Here is an https://support.google.com/nexus/answer/2840804?hl=en[example guide for Google Nexus devices]. It may be different for other devices. Or, use the link:https://developer.android.com/studio/command-line/adb[Android Debug Bridge^](adb) push command to push the certificate to the connected device:
+* Email attachment (Recommended) - Email the certificate to an address accessible from the device and download the attachment.
+* Cloud service - Use a cloud storage service such as Dropbox or Google Drive which is accessible from the device, and download from the device.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/proc_self-signed-certs.adoc]: is "_that_ is accessible from the device. " the correct grammar in here?

File: modules/ROOT/pages/_partials/proc_self-signed-certs.adoc [hunk 2]
```diff
@@ -43,77 +42,91 @@ The resulting file is located in the `/tmp` directory.
...
+* USB - Attach the device to the machine using USB and drag the certificate to a devices file system.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/proc_self-signed-certs.adoc]: "using a USB"

File: modules/ROOT/pages/_partials/proc_self-signed-certs.adoc [hunk 3]
```diff
@@ -43,77 +42,91 @@ The resulting file is located in the `/tmp` directory.
...
-certificate and let you install it. Here is https://support.google.com/nexus/answer/2844832?hl=en[a sample guide for Google Nexus devices].
+.. Go to Settings > Security & location > Advanced > Encryption & credentials > Install from SD card.
+You are prompted to choose a file.
+.. Navigate to the `Downloads` folder and choose the certificate file.
+The Android system detects the certificate and lets you install it. For more information, see the link:https://support.google.com/nexus/answer/2844832?hl=en[sample guide for Google Nexus devices].
 
-A simple way to test if the self-signed certificate has been installed correctly is to open the OpenShift web console in a browser (like Google Chrome) on the device and you should not get any warnings or errors about the certificate.
+To verify the self-signed certificate is installed correctly, use a browser on the deveice to open the OpenShift web console. You should not see any warnings or errors about the certificate.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/proc_self-signed-certs.adoc]:
```suggestion
To verify the self-signed certificate is installed correctly, use a browser on the device to open the OpenShift web console. You should not see any warnings or errors relating to the certificate.
```

File: modules/ROOT/pages/_partials/proc_self-signed-certs.adoc (hunk 1 above)
Comment (finp) [reply modules/ROOT/pages/_partials/proc_self-signed-certs.adoc]: yep, should be 'that'

File: modules/ROOT/pages/_partials/proc_self-signed-certs.adoc (hunk 2 above)
Comment (finp) [reply modules/ROOT/pages/_partials/proc_self-signed-certs.adoc]: removed, the point it to 'attach', USB is already mentioned.

**Pull request merged**